packages/miniflare/test/http/response.spec.ts

```diff
@@ -1,60 +1,57 @@
-import test from "ava";
 import { Response, WebSocketPair } from "miniflare";
+import { expect, test } from "vitest";
 
-test("Response: static methods return correctly typed values", async (t) => {
+test("Response: static methods return correctly typed values", async () => {
 	const error = Response.error();
 	// noinspection SuspiciousTypeOfGuard
-	t.true(error instanceof Response);
+	expect(error).toBeInstanceOf(Response);
 
 	const redirect = Response.redirect("http://localhost/", 302);
 	// noinspection SuspiciousTypeOfGuard
-	t.true(redirect instanceof Response);
-	t.is(redirect.status, 302);
-	t.is(redirect.headers.get("Location"), "http://localhost/");
+	expect(redirect).toBeInstanceOf(Response);
+	expect(redirect.status).toBe(302);
+	expect(redirect.headers.get("Location")).toBe("http://localhost/");
 
 	const json = Response.json({ testing: true }, { status: 404 });
 	// noinspection SuspiciousTypeOfGuard
-	t.true(json instanceof Response);
-	t.is(json.status, 404);
-	t.is(json.headers.get("Content-Type"), "application/json");
-	t.deepEqual(await json.json(), { testing: true });
+	expect(json).toBeInstanceOf(Response);
+	expect(json.status).toBe(404);
+	expect(json.headers.get("Content-Type")).toBe("application/json");
+	expect(await json.json()).toEqual({ testing: true });
 });
 
-test("Response: requires status 101 for WebSocket handshakes response", (t) => {
+test("Response: requires status 101 for WebSocket handshakes response", () => {
 	const pair = new WebSocketPair();
-	t.throws(() => new Response(null, { webSocket: pair["0"] }), {
-		instanceOf: RangeError,
-		message: "Responses with a WebSocket must have status code 101.",
-	});
+	expect(() => new Response(null, { webSocket: pair["0"] })).toThrow(
+		"Responses with a WebSocket must have status code 101."
+	);
 });
-test("Response: only allows status 101 for WebSocket response", (t) => {
-	t.throws(() => new Response(null, { status: 101 }), {
-		instanceOf: RangeError,
-		message: 'init["status"] must be in the range of 200 to 599, inclusive.',
-	});
+test("Response: only allows status 101 for WebSocket response", () => {
+	expect(() => new Response(null, { status: 101 })).toThrow(
+		'init["status"] must be in the range of 200 to 599, inclusive.'
+	);
 });
 
-test("Response: clone: returns correctly typed value", async (t) => {
+test("Response: clone: returns correctly typed value", async () => {
 	const response = new Response("text");
 	const clone1 = response.clone();
 	const clone2 = clone1.clone(); // Test cloning a clone
 
 	// noinspection SuspiciousTypeOfGuard
```

Comment:
I think you can remove these comments now that we are not using `instanceof` directly.